src/main/java/ua/hillel/FileManager.java

```diff
@@ -0,0 +1,107 @@
+package ua.hillel;
+
+import java.io.*;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+public class FileManager {
+
+    private static final Logger LOGGER = Logger.getLogger(FileManager.class.getName());
+    private String currentDirectory;
+
+    public FileManager() {
+        this.currentDirectory = "/";
+    }
+
+    public void start() {
+        while (true) {
+
+            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+            String rawCommand;
+            try {
+                rawCommand = br.readLine();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            Command command = parseCommand(rawCommand);
+
+            if (command != null) {
+                switch (command) {
+                    case CD -> changeDirectory(rawCommand);
+                    case LS -> listDirectory();
+                    case CP -> copyFiles(rawCommand);
+                    case PWD -> printWorkDirectory();
+                }
+            }
+        }
+    }
+
+    private Command parseCommand(String commandRaw) {
+        try {
+            if (commandRaw.contains(" ")) {
+                String subCommand = commandRaw.split(" ")[0];
+                return Enum.valueOf(Command.class, subCommand.toUpperCase());
+            } else {
+                return Enum.valueOf(Command.class, commandRaw.toUpperCase());
+            }
+
+        } catch (Exception ignore) {
+        }
```

Comment:
It would be better to log an exception message here to understand what went wrong